server/storage/mvcc/key_index_test.go

```diff
@@ -687,6 +707,10 @@ func TestGenerationWalk(t *testing.T) {
 	}
 }
 
+// TODO(fuweid):
+//
+// Replace {14, 1} with {15, 1} because ETCD server doesn't allow duplicate
+// keys in one txn.
```

Comment:
Can you fix this firstly so that we have meaningful test cases?
Please let me know if you need any help on this.

Reply:
Please do it in a separate PR.